src/DatePicker/datePicker.scss

```diff
@@ -1,224 +1,246 @@
 @import "../utils/theme/variables.scss";
+@import "node_modules/react-dates/lib/css/_datepicker";
 
-$react-dates-width-input: 150px !default;
-$react-dates-color-primary: $brc-azure !default;
-
-$react-dates-color-primary-shade-1: $brc-azure30 !default; // used as border of selected range
-$react-dates-color-primary-shade-2: $brc-azure10 !default; // used as background of selected range
-$react-dates-color-primary-shade-3: $brc-azure !default; // used as border on hover range
-$react-dates-color-primary-shade-4: $brc-azure30 !default; // used as background of hover range
-
-$react-dates-color-secondary: $brc-darkGreyBlue !default; // font-color on hover range
-$react-dates-color-white: $brc-white !default;
-$react-dates-color-gray: $brc-darkGreyBlue !default;
-$react-dates-color-gray-dark: darken($react-dates-color-gray, 10.5%) !default;
-$react-dates-color-gray-light: $brc-coolGrey;
-$react-dates-color-gray-lighter: lighten($react-dates-color-gray, 45%) !default;
-$react-dates-color-gray-lightest: lighten(
-  $react-dates-color-gray,
-  60%
-) !default;
-$react-dates-color-highlighted: #ffe8bc !default;
-
-$react-dates-color-border: $brc-silver !default;
-$react-dates-color-border-light: $brc-silver !default;
-$react-dates-color-border-medium: $brc-silver !default;
-$react-dates-color-placeholder-text: $brc-coolGrey !default;
-$react-dates-color-text: $brc-darkGreyBlue !default;
-$react-dates-color-text-focus: $brc-darkGreyBlue !default;
-$react-dates-color-focus: transparent !default;
-
-$react-dates-z-index: 100 !default;
-
-@import "~react-dates/css/variables.scss";
-@import "~react-dates/css/styles.scss";
+$width-input: 150px !default;
+$month-header-background: $brc-azure !default;
+$month-header-color: $brc-white !default;
+
+$z-index: 100 !default;
 
 $input-background: $brc-gradientGrey !default;
 $input-background-hover: $brc-gradientDarkGrey !default;
 $input-background-open: $brc-gradientDarkGrey !default;
+$input-border-color: $brc-silver !default;
+$input-border-color-focused: $brc-azure !default;
+
+$default-day-border: $brc-silver !default;
+$default-day-background: $brc-paleGrey !default;
+$default-day-background-hover: $brc-cloudyGrey !default;
+$default-day-color: $brc-darkGreyBlue !default;
+$default-day-color-hover: $brc-darkGreyBlue !default;
+$blocked-day-background: $brc-white !default;
+$blocked-day-color: $brc-coolGrey !default;
+$selected-day-background: $brc-azure !default;
+$selected-day-color: $brc-darkGreyBlue !default;
+$selected-day-hover-background: $brc-azure !default;
+$selected-day-hover-color: $brc-darkGreyBlue !default;
+$hover-interval-day-background: $brc-azure10 !default;
+$hover-interval-day-border: $brc-azure !default;
+$hover-interval-day-color: $brc-darkGreyBlue !default;
+$selected-interval-day-background: $brc-azure30 !default;
+$selected-interval-day-border: $brc-azure10 !default;
+$selected-interval-day-color: $brc-darkGreyBlue !default;
+$outside-day-background: $brc-white !default;
```

Comment:
as discussed offline, I would normalize the naming of variables to have the `-hover` modifier always at the end + adding a prefix/namespace for the single component (`date-picker-<variable-name>`)